cmd/burrow/commands/configure.go

```diff
@@ -16,8 +18,23 @@ import (
 	logging_config "github.com/hyperledger/burrow/logging/config"
 	"github.com/hyperledger/burrow/logging/config/presets"
 	"github.com/jawher/mow.cli"
+	"github.com/tendermint/go-amino"
+	tm_crypto "github.com/tendermint/go-crypto"
+	"github.com/tendermint/tendermint/p2p"
 )
 
+func processTemplate(pkg *deployment.Config, config *genesis.GenesisDoc, templateIn, templateOut string) error {
```

Comment:
minor but can we put this at end, I like to see most important methods first

Reply:
Fixed